incubator/zookeeper/templates/pdb.yaml

```diff
@@ -1,18 +1,18 @@
 apiVersion: policy/v1beta1
 kind: PodDisruptionBudget
 metadata:
-  name: "{{ printf "pd-zk-%s" .Release.Name | trunc 24 }}"
```

Comment:
Probably can factor out the `.Release.Name | trunc 24` into a separate template file like: https://github.com/kubernetes/charts/blob/master/incubator/mongodb-replicaset/templates/_helpers.tpl

Might be cleaner that way. Also should be trunc 63 I think.